docs/migrate-to-antrea.md

```diff
@@ -0,0 +1,73 @@
+# Migrate from another CNI to Antrea
+
+This document describes how to migrate from other CNIs to Antrea.
+
+NOTE: The following is a reference list of CNIs and versions we have already
+verified the migration process. CNIs and versions that are not listed here
+might also work. Please create an issue if you run into problems during the
+migration to Antrea. During the migration process, no Kubernetes resources
+should be created or deleted, otherwise the migration process might fail or
+some unexpected problems might occur.
+
+| CNI     | Version |
+|---------|---------|
+| Calico  | v3.26   |
+| Flannel | v0.22.0 |
+
+The migration process is divided into three steps:
+
+1. Clean up the old CNI.
+2. Install Antrea in the cluster.
+3. Restart all Pods in the cluster in-place.
```

Comment:
```suggestion
3. Deploy Antrea migrator.
```
We can talk about "restart all Pods in the cluster in-place" in its own section.